packages/ui/src/components/va-checkbox/VaCheckbox.vue

```diff
@@ -128,9 +129,13 @@ export default defineComponent({
         color: computedError.value ? getColor('danger') : (props.success ? getColor('success') : ''),
         padding: !props.label
           ? ''
-          : props.leftLabel
-            ? '0 0.5rem 0 0'
-            : '0 0 0 0.5rem',
+          : props.vertical
+            ? '0.3rem 0 0.3rem 0.5rem'
```

Comment:
We will need to move these values to CSS variables under `_variables.scss`, so they can be configured.